src/test/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/impl/FieldMapperTest.java

```diff
@@ -30,49 +30,53 @@
  */
 class FieldMapperTest {
 
-//    private static final String delimiter = random(); // TODO flip this back on when escaping is implemented
+    //    private static final String delimiter = random(); // TODO flip this back on when escaping is implemented
```

Comment:
not really appropriate for this PR, but do we want a test with randomness? this may fail once then pass... (also, if we were to keep the "random" method, I'd suggest renaming it...)

Reply:
The point is it should work no matter what the random sequence is.  I'm always open to naming changes because I suck at naming.  Let me know what you think.